### PR TITLE
Bugfix/curator tracklist

### PR DESCRIPTION
**Pull request by mihaelaaleks**

### CODE REVIEW

File: plotly_flask/routes.py [hunk 1]
```diff
@@ -56,8 +56,8 @@ def curator():
 
         spotify = curator_logic.create_spotify()
         track_recommendations = curator_logic.get_cleaned_recommendations(
-            limit=50,
             spotify=spotify,
+            limit=50,
```
Comment (LeoFrancoCrespo) [inline plotly_flask/routes.py:60]: What is this limit? Is it the number of recommendations limited

I'd add this to the top of this file after the imports into a `N_RECOMMENDATION_LIMIT = 50` and then call this as `limit = N_RECOMMENDATION_LIMIT`

File: plotly_flask/models/curator_logic.py [hunk 2]
```diff
@@ -40,71 +47,87 @@ def get_recommendation(limit:int,
         target_<attr>: audio features for generating recommendation
 
     Returns:
-        dict: Json response 
+        dict: Json response
     """
-    s_artists = ['6XYvaoDGE0VmRt83Jss9Sn', '07b9qW7pabKGO29JPWXn9m']
+    s_artists = ["6XYvaoDGE0VmRt83Jss9Sn", "07b9qW7pabKGO29JPWXn9m"]
     if limit >= 100:
```
Comment (LeoFrancoCrespo) [inline plotly_flask/models/curator_logic.py:53]: Similar to the other comment, I'd include this as some top level constant
``` 
N_MAX_RECOMMENDATION_LIMIT = 100
```

and then you just call that here.

I like having constants for these things :D

**Review by LeoFrancoCrespo — APPROVED** [review]
Looks functional to me, nothing to block the PR from going forward.

I do suggest two places with hard coded values defining constants, but it's not that big a deal.

File: plotly_flask/routes.py (hunk 1 above)
Comment (mihaelaaleks) [reply plotly_flask/routes.py:60]: Good idea! My next PR is gonna be fleshing out curator behaviors, so I'll fix it there.

**Pull request merged**